docs/source/authentication.rst

```diff
@@ -76,7 +76,7 @@ repo. The following is a sample auth section in the config file for the PAM back
     use_ssl = True
     cert = /path/to/ssl/cert/file
```

Comment:
Thought about it; decided not to. This SSL cert could be different to the default self-signed cert.